modules/ossm-release-notes-supported-versions.adoc

```diff
@@ -33,5 +31,29 @@ See this following table to know supported versions in {SMProduct} 3.
 | {istio-latest} ^[1]^
 |===
 
+See this following table to know supported versions in {SMProduct} 3.
+
+== {SMProduct} 3.0 supported versions
+
+[cols="1,1"]
+|===
+| Feature | Supported versions
+
+|{SMProduct} 3 Operator
+|3.0
+
+|{SMProduct} `Istio` control plane resource
+|1.24.3
```

Comment:
1.24.4

Reply:
Hi @pbajjuri20! the changes you have requested are for 3.0 and the istio version then was 1.23.3. I have removed the attributes from the 3.0.1 table to avoid confusion. PTAL!

Reply:
@pbajjuri20! Kindly give your approval if everything is okay! :)